gui/app/periodic_graph_painter.cpp

```diff
@@ -0,0 +1,77 @@
+#include "periodic_graph_painter.h"
```

Comment:
Unfortunately I wasn't able to see how this looks because in the current version of the PR when I launch it I get a stream of errors:

qrc:/modes/CommandPressureMode.qml:114:17: Unable to assign QJSValue to QVector<QPointF>
qrc:/modes/CommandPressureMode.qml:133:17: Unable to assign QJSValue to QVector<QPointF>
qrc:/modes/CommandPressureMode.qml:156:17: Unable to assign QJSValue to QVector<QPointF>
... (repeats indefinitely) ...

Reply:
That is weird. I tested on the rPI and on Mac and it works fine. Can you tell me which version of Qt you are running?